src/CodeTracking.jl

```diff
@@ -251,7 +256,7 @@ function definition(::Type{Expr}, method::Method)
             end
         end
     end
-    return def === nothing || ismissing(def) ? nothing : copy(def[2])
+    return def === nothing || ismissing(def) ? nothing : copy(def[end][2])
```

Comment:
Do you think its easier to read if we put the condition in brackets?
```suggestion
    return (def === nothing || ismissing(def)) ? nothing : copy(def[end][2])
```

or maybe flip the order
```suggestion
    return ismissing(def)  || def === nothing ? nothing : copy(def[end][2])
```

If we had a Compat.jl dependency we could have
```suggestion
    return ismissing(def)  || isnothing(def) ? nothing : copy(def[end][2])
```

Reply:
Nah, I should have used `def === missing` too. No risk of runtime dispatch that way.